Improve docstrings.

vic opened the pull request:
Code should be 4-space indented to look right on the generated html.
Use the ~S sigil convention for docs.

vic committed:
Improve docstrings.

Code should be 4-space indented to look right on the generated html.
Use the ~S sigil convention for docs.

aleandros commented:
Thanks for the intentation corrections! It was killing me and last night I was to tired to keep going.
I have an issue with adding the ~S sigil to every docstring, even those that don't do any string interpolation.

The elixir community style guide https://github.com/levionessa/elixir_style_guide#documentation doesn't recommend it as mandatory.

Commit:
Removed ~S sigil where not necessary

I went through the style guide and the documentation of phoenix and
elixir itself, and it didn't seem as common to use the ~S sigil for
docstrings that did not require it.

aleandros reviewed (approved):
I won't stagnate the discussion any further. Accepted the changes to corrected indendation (thanks!) but opted for removing not-required ~S sigils.
